.github/workflows/Test.yml

```diff
@@ -484,7 +484,9 @@ jobs:
           pip3 install -r testsuite/requirements.txt
 
       - name: '🚦 Test installation of pyGHDL through pip'
-        run: pip install ("git+https://github.com/ghdl/ghdl.git@" + $env:GHDL_HASH)
+        run: |
+          pip install pyTooling pyTooling.Packaging wheel
```

Comment:
As discussed in a meeting, this is not required since `pyproject.toml` was added.